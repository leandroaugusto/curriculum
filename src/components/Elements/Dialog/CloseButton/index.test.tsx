import { render } from "@testing-library/react";
import DialogCloseIcon from ".";

describe("DialogCloseIcon", () => {
  it("renders the DialogCloseIcon element", () => {
    const { container } = render(<DialogCloseIcon />);

    expect(container).toMatchSnapshot();
  });
});
