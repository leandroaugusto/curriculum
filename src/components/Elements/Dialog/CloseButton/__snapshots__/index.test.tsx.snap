// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogCloseIcon renders the DialogCloseIcon element 1`] = `
<div>
  <button
    class="bg-white rounded-md p-2 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500 absolute right-0 top-0"
    type="button"
  >
    <span
      class="sr-only"
    >
      Close menu
    </span>
    <svg
      aria-hidden="true"
      class="h-6 w-6"
      fill="none"
      stroke="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M6 18L18 6M6 6l12 12"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
  </button>
</div>
`;
